plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4350,6 +4350,40 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
                 brushableBlock.setItem(item.getItemStack());
                 brushableBlock.update();
             });
+
+            // <--[tag]
+            // @attribute <LocationTag.waxed>
+            // @returns ElementTag(Boolean)
+            // @mechanism LocationTag.waxed
+            // @group world
+            // @description
+            // If the location is a sign block, returns whether it is waxed (locked to prevent players editing the text).
+            // -->
+            tagProcessor.registerTag(ElementTag.class, "waxed", (attribute, object) -> {
+                if (!(object.getBlockStateForTag(attribute) instanceof Sign sign)) {
+                    attribute.echoError("Location is not a valid Sign block.");
+                    return null;
+                }
+                return new ElementTag(sign.isWaxed());
+            });
+
+            // <--[mechanism]
+            // @object LocationTag
+            // @name waxed
+            // @input ElementTag(Boolean)
+            // @description
+            // Sets whether the sign at the location is waxed.
```

Comment:
Should have the same `(locked to prevent players editing the text)` clarification the tag has.